sagemaker-python-sdk/tensorflow_script_mode_quickstart/tensorflow_script_mode_quickstart.ipynb

```diff
@@ -58,7 +58,13 @@
    "source": [
     "## Preparing the training script\n",
     "\n",
-    "Let's start by cloning the repository that contains the example:"
+    "For training scripts, let's use Git integration for SageMaker Python SDK here. That is, you can specify a training script that is stored in a GitHub (or other Git) or CodeCommit repository as the entry point for the estimator, so that you don't have to download the scripts locally. If you do so, source directory and dependencies should be in the same repo if they are needed.\n",
```

Comment:
That is, you can specify a training script that is stored in a GitHub, CodeCommit, or other Git repository as the entry point for the estimator.